packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -103,12 +104,18 @@ const Watchlist = () => {
     }
   };
 
+  const setPreviewMode = mode => {
+    mode === 'CHART' && setDisplayPreview(true);
+    mode === 'DELETE' && setDisplayDelete(true);
```

Comment:
From the above, this could be a ternary:
`mode === 'CHART'? setDisplayWindow(SOME_CONSTANT_CHART) : setDisplayWindow(SOME_CONSTANT_DELETE)`?

Reply:
Changed due to refactor.